package.json

```
@@ -62,6 +62,7 @@
     "algoliasearch-helper": "^3.10.0",
     "classnames": "^2.2.5",
     "hogan.js": "^3.0.2",
+    "htm": "^3.1.1",
```

We could loosen the dep range so that the risk of getting duplicated is lower?

```suggestion
    "htm": "^3.0.0",
```

---

Good idea! Addressed in 610ef47b.